connections/oikotie/oikotie_mapper.py

```diff
@@ -367,6 +367,20 @@ def map_new_development_status(
         )
 
 
+def form_description(elastic_apartment):
+    """
+    Fetch link to apartment presentation and add it to the end of project description
+    """
+    main_text = getattr(elastic_apartment, "project_description", None)
+    link = getattr(elastic_apartment, "url", None)
+    link_message = "Tarkempi esittely Helsingin kaupungin asuntotuotannon sivulla"
+
+    if main_text and link:
+        return f"{main_text}\n{link_message} {link}"
+    else:
+        return main_text
```

Comment:
Should it send a link if the main text hasn't been set? Ex. like this?
```
link_text =  f"{link_message} {link}" if link else None

if main_text or link_text:
    return "\n".join(filter(None, [main_text, link_text]))
return None

```

The same applies also to the Etuovi side.

Reply:
Not sure... Should this be agreed with HKI that description can be empty?

Reply:
Previously the description was nullable in the code, so it should be already agreed. But it is good to make sure that if no description is set, then we will still put the link to go if that exists.